src/test/librbd/test_main.cc

```diff
@@ -23,12 +24,28 @@ extern void register_test_mirroring_watcher();
 extern void register_test_object_map();
 extern void register_test_operations();
 extern void register_test_trash();
+#if defined(WITH_RBD_RWL)
+extern void register_test_blockguard();
+extern void register_test_write_log_map();
+#endif //defined(WITH_RBD_RWL)
 #endif // TEST_LIBRBD_INTERNALS
 
 int main(int argc, char **argv)
 {
   setenv("RBD_FORCE_ALLOW_V1","1",1);
-
+#if defined(WITH_RBD_RWL)
+  /* Disable actual PMDK persistence guarantees for unit tests */
+  setenv("PMEM_IS_PMEM_FORCE","1",1);
+//#else
```

Comment:
Nit: remove commented out code

Reply:
We have already set the option, so remove it here.